src/net_processing.cpp

```diff
@@ -563,41 +561,6 @@ static bool MarkBlockAsInFlight(CTxMemPool& mempool, NodeId nodeid, const uint25
     return true;
 }
 
-/** Check whether the last unknown block a peer advertised is not yet known. */
```

Comment:
In commit "net_processing: Move some static functions to PeerManager" (053c84ce5a99f0ef87fe23164dc11a0eac725680)

Might be nice to split this commit, and move the function bodies in a MOVEONLY commit. But this is maybe less important than it used to be now that git diff displays moved code better than it used to